app/uk/gov/hmrc/estates/controllers/RegisterEstateController.scala

```diff
@@ -44,11 +45,11 @@ class RegisterEstateController @Inject()(desService: DesService, config: AppConf
   def registration(): Action[JsValue] = identifierAction.async(parse.json) {
     implicit request =>
 
-      val registrationJsonString = request.body.toString()
+      val payload = request.body.applyRules().toString
 
       validationService
         .get(config.estatesApiRegistrationSchema)
-        .validate[EstateRegistration](registrationJsonString) match {
+        .validate[EstateRegistration](payload) match {
```

Comment:
This would do as intended as the frontend isn't going to be sending the pull payload for register. The backend will be building the document from the transforms. We need to remove it then at the transform level.